src/js/api/order.js

```diff
@@ -238,6 +238,11 @@ ripe.Ripe.prototype.attachmentOrderP = function(number, attachmentId, options) {
     });
 };
 
+ripe.Ripe.prototype.getAttachmentOrderUrl = function(key, options = {}) {

```

Comment:
Same approach as `getMeshUrl` and `getLogoUrl`:
- no buildQuery
- _getAttachmentOptions

Also, this is in the wrong file (order.js) and should be in attachment.js (look at the endpoint you're using).